net/api/impl/posix.h

```diff
@@ -86,8 +88,8 @@ class HTTPClientPOSIX final {
       // Don't uncomment the next line!
       // connection.SendEOF();
       message_.reset(new HTTPRedirectableReceivedMessage(connection));
-      // TODO(dkorolev): Rename `URL()`, it's only called so now because of HTTP request/response format.
-      const int response_code_as_int = atoi(message_->URL().c_str());
+      // TODO(dkorolev): Rename `Path()`, it's only called so now because of HTTP request/response format.
+      const int response_code_as_int = atoi(message_->Path().c_str());
       response_code_ = static_cast<HTTPResponseCode>(response_code_as_int);
       if (response_code_as_int >= 300 && response_code_as_int <= 399 && !message_->location.empty()) {
```

Comment:
(It would actually make things easier, but, for some strange reason, I thought that parsing all the headers into a hash-map would be undesirable for a lightweigh server, especially given that there already is a mechanism to only expose the ones we need.)